apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/gradingPanel.ejs

```diff
@@ -0,0 +1,101 @@
+<form name="manual-grading-form" method="POST"
+      data-max-auto-points="<%= assessment_question.max_auto_points %>"
+      data-max-manual-points="<%= assessment_question.max_manual_points %>"
+      data-max-points="<%= assessment_question.max_points %>"
+      data-rubric-active="<%= !!locals.rubric_data %>"
+      data-rubric-replace-auto-points="<%= locals.rubric_data?.replace_auto_points %>"
+      data-rubric-starting-points="<%= locals.rubric_data?.starting_points %>"
+      data-rubric-max-extra-points="<%= locals.rubric_data?.max_extra_points %>"
+      data-rubric-min-points="<%= locals.rubric_data?.min_points %>">
+  <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+  <input type="hidden" name="modified_at" value="<%= instance_question.modified_at %>">
+  <input type="hidden" name="submission_id" value="<%= submission.id %>">
+  <ul class="list-group list-group-flush">
+    <li class="list-group-item">
+      <div class="form-group row justify-content-center">
+        <label class="custom-control-inline col-auto mx-0">
+          <span class="">Points</span>
+          <div class="custom-control custom-switch mx-2">
+            <input class="custom-control-input js-manual-grading-pts-perc-select"
+                   name="use_score_perc" type="checkbox">
+            <span class="custom-control-label">Percentage</span>
+          </div>
+        </label>
+      </div>
+    </li>
+    <% if (!assessment_question.max_auto_points && !(locals.auto_points ?? instance_question.auto_points)) { %>
+    <li class="list-group-item">
+      <%- include('manualPointsSection'); %>
+      <%- include('rubricInputSection'); %>
+    </li>
+    <% } else { %>
+    <li class="list-group-item">
+      <%- include('manualPointsSection'); %>
+      <% if (!locals.rubric_data?.replace_auto_points) { %><%- include('rubricInputSection'); %><% } %>
```

Comment:
At the moment the "Rubric" button that lets us edit the rubric can end up far away from the rubric itself (see image). We could put the auto points above the manual points (they do come first, temporally) and we could also shift the "Rubric" button to be right next to the rubric items if we have a rubric (rather than being next to the manual grading points)?

<img width="552" alt="image" src="https://github.com/PrairieLearn/PrairieLearn/assets/875833/90064123-1f3a-4738-9160-a2667f87d537">

Reply:
Moved to total points in 50d5a9d96.